Documentation/ContentObjects/Hmenu/Language.rst

```diff
@@ -1,90 +1,78 @@
 
-.. seealso::
-   The :ref:`LanguageMenuProcessor <LanguageMenuProcessor>` can be used to
-   create a language menu styled by Fluid.
+..  seealso::
+    The :ref:`LanguageMenuProcessor <LanguageMenuProcessor>` can be used to
+    create a language menu styled by Fluid.
 
 Creates a language selector menu.
 
 The :typoscript:`special = language` type will create menu items based on
 the current page record but with the language record for each language
 overlaid if available.
 
-.. contents::
-   :local:
+..  contents::
+    :local:
+
+..  _hmenu-special-language-properties:
 
 Properties
 ==========
 
+..  _hmenu-special-language-value:
 
-.. ### BEGIN~OF~TABLE ###
-
-.. _hmenu-special-language-value:
-
-.. container:: table-row
-
-   Property
-         special.value
-
-   Data type
-         comma list of sys\_language uids /:ref:`stdWrap <stdwrap>` or `auto`
-
-   Description
-         The number of elements in this list determines the number of menu
-         items. Setting to `auto` will include all available languages from
-         the current site.
-
-.. _hmenu-special-language-normalwhennolanguage:
-
-
-.. container:: table-row
+special.value
+-------------
 
-   Property
-         special.normalWhenNoLanguage
+..  confval:: special.value
+    :name: hmenu-language-special-value
+    :type: comma separated list of language UIDs /:ref:`stdWrap <stdwrap>` or `auto`
 
-   Data type
-         boolean
+    The number of elements in this list determines the number of menu
+    items. Setting to `auto` will include all available languages from
+    the current site.
 
-   Description
-         If set to 1, the button for a language will be rendered as a non-
-         disabled button even if no translation is found for the language.
+..  _hmenu-special-language-normalwhennolanguage:
 
+special.normalWhenNoLanguage
+----------------------------
 
-.. ###### END~OF~TABLE ######
 
-[tsref:(cObject).HMENU.special = language]
+..  confval:: special.normalWhenNoLanguage
+    :name: hmenu-language-special-normalWhenNoLanguage
+    :type: boolean
 
+    If set, the button for a language will be rendered as a non-
+    disabled button even if no translation is found for the language.
 
-.. _hmenu-special-language-examples:
 
-Example
-========
+..  _hmenu-special-language-examples:
 
-Creates a language menu with pure Typoscript:
+Example: Create a language menu with pure TypoScript
+====================================================
 
-.. include:: /CodeSnippets/Menu/TypoScript/LanguageMenuLib.rst.txt
+..  include:: /CodeSnippets/Menu/TypoScript/LanguageMenuLib.rst.txt
 
-.. seealso::
+..  seealso::
    For a language menu styled by Fluid see
    :ref:`LanguageMenuProcessor <LanguageMenuProcessor>`.
 
 Note on item states
 ====================
 
-When "TSFE->sys\_language\_uid" matches the sys\_language uid for an
-element the state is set to "ACT", otherwise "NO". However, if a page
+When `TSFE->sys_language_uid` matches the UID of an
```

Comment:
TSFE->sys_language_uid was deprecated in 9.4 and removed with 10.0:
https://github.com/TYPO3/typo3/blob/9.5/typo3/sysext/frontend/Classes/Controller/TypoScriptFrontendController.php#L603-L611

So, this sentence should be adjusted.

Reply:
You have any idea, how it works now?

Reply:
Nope, maybe add a todo and an issue.

Reply:
maybe like this